.eslintrc

```diff
@@ -91,6 +91,8 @@
       0,
       "double"
     ],
+    "react/jsx-uses-react": 2,
+    "react/react-in-jsx-scope": 2,
```

Comment:
Nice.  Will these need to change at all when the [eslint upgrade](https://github.com/redfin/react-server/pull/71) lands?